src/app/containers/DashboardView/DashboardView.tsx

```diff
@@ -0,0 +1,103 @@
+import _ from 'lodash'
+import React from 'react'
+import { observer, inject } from 'mobx-react'
+import { Redirect, Route, RouteComponentProps } from 'react-router-dom'
+import { Box, Flex } from 'reflexbox'
+import { PortfolioView } from '../PortfolioView'
+import { Button } from '../../components'
+
+export interface IProps extends InjectedProps, RouteComponentProps<null> {
+}
+
+interface InjectedProps {
+  userStore?: UserStore
+  hasLoadedState: boolean
+}
+
+@inject((store: RootStore): InjectedProps => ({
+  userStore: store.user,
+  hasLoadedState: store.user.hasLoadedState,
+}))
+@observer
+class DashboardView extends React.Component<IProps, {}> {
+
+  componentWillReceiveProps(nextProps: IProps) {
+    if (nextProps.hasLoadedState && !this.props.hasLoadedState) {
+      const pathname = this.props.location.pathname
+      // todo: refactor - store selected portfolio id in the state maybe?
+      if (pathname.split('/').length === 2) {
+        const firstPortfolio = _.first(this.props.userStore!.portfolios)
+        if (firstPortfolio) {
```

Comment:
Mon tai neparenk to portfolio pirmo, kai atvarai po logino i `/dashboard`, rodo tik ton dashboarda ir blank page. Nzn kami beda :/

Reply:
hmm, reik tvarkyt tada. Ci sep mata toks laikens sprendems su tuo routu, reiketu geriau state laikyt